packages/surface/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@hig/surface",
+  "version": "0.1.0",
+  "description": "HIG Surface component",
+  "author": "Autodesk Inc.",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Autodesk/hig.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "build/index.js",
+  "module": "build/index.es.js",
+  "files": [
+    "build/*"
+  ],
+  "dependencies": {
+    "@hig/theme-context": "^2.1.0",
+    "emotion": "^10.0.0",
+    "prop-types": "^15.7.1"
+  },
+  "devDependencies": {
+    "@hig/babel-preset": "^0.1.1",
+    "@hig/eslint-config": "^0.1.0",
+    "@hig/jest-preset": "^0.1.0",
+    "@hig/rich-text": "^1.0.0",
+    "@hig/scripts": "^0.1.2",
+    "@hig/semantic-release-config": "^0.1.0",
+    "@hig/typography": "^1.0.1"
```

Comment:
`1.0.2` is the latest for `typography`.

Reply:
Yah true on the version, but I think that's how we get those `theme-context` errors locally as if we don't dump node modules and already have the older version, then it won't install the newer one which could be dependent on the newer `theme-context`/`theme-data`. I think it's not as much of an issue if you're installing fresh.